test-configs.yaml

```diff
@@ -165,6 +170,12 @@ test_plans:
     filters:
       - passlist: {defconfig: ['kselftest']}
 
+  ltp:
+    rootfs: debian_buster-ltp_nfs
+    pattern: 'ltp/{category}-{method}-{protocol}-nfs-ltp-template.jinja2'
```

Comment:
With the suggestion above to use `root_type: nfs`
```suggestion
```

Reply:
Added above change